app/routes.py

```diff
@@ -1,2 +1,345 @@
-from flask import Blueprint
+from app import db
+from flask import request, Blueprint, jsonify
+from .models.task import Task
+from .models.goal import Goal
+import datetime
+import requests
+import os
+
+
+### WAVES 1 AND 2 - CRUD for TASKS
+
+# Create an endpoint specifically for tasks (all the routes start with /tasks):
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def is_int(value):
+    try:
+        return int(value)
+    except ValueError:
+        return False
+
+
+# define a route with default empty string to GET all tasks:
+@tasks_bp.route("", methods=["GET"], strict_slashes=False)
+def get_tasks():
+    sort_query = request.args.get("sort")
+    if sort_query == "asc":
+        tasks = Task.query.order_by("title")
+    elif sort_query == "desc":
+        tasks = Task.query.order_by(Task.title.desc())  
+    else:
+        tasks = Task.query.all()
+
+    tasks_response = []
+
+    for task in tasks:
+        if task.completed_at == None:
+            tasks_response.append({
+            "id": task.id,
+            "title": task.title,
+            "description": task.description,
+            "is_complete": False
+        })
+        else:
+            tasks_response.append({
+            "id": task.id,
+            "title": task.title,
+            "description": task.description,
+            "is_complete": True
+            })
```

Comment:
This would have been a good spot to use your `Task` helper method `to_dict()`. 
```suggestion
    for task in tasks:
            tasks_response.append(task.to_dict())
```